src/plugins/dashboard/public/dashboard_container/embeddable/create/create_dashboard.ts

```diff
@@ -504,5 +477,32 @@ export const initializeDashboard = async ({
     );
   });
 
+  // --------------------------------------------------------------------------------------
```

Comment:
Controls setup re-defines `filters$`. If we call `startDashboardSearchSessionIntegration` before setting up controls then the newSession$ will subscribe to old `filter$`and not get updates input$.filters changes.